routes.yaml

```diff
@@ -0,0 +1,6 @@
+app_name: chips-rest-interfaces-consumer
+group: internalapi
+
+weight: 900
+routes:
+  1: ^/company/(.*)/chips-rest-interfaces-consumer
```

Comment:
We do not need /company/(.*) as the only endpoint on our service will be the healthcheck endpoint.
As the only endopint will be a healthcheck endpoint. As normal flow won't involve using rest.

Reply:
"company" removed